manticore/platforms/linux.py

```diff
@@ -2621,8 +2632,18 @@ def solve_to_fd(data, fd):
             if name in ('_receive', '_read') and fd == 0:
                 solve_to_fd(data, inn)
 
+        for a in self.argv:
+            solve_to_fd(a, argIO)
+            argIO.write("\n")
+
+        for e in self.envp:
+            solve_to_fd(e, envIO)
+            envIO.write("\n")
```

Comment:
Would this label the variables correctly in the output?

Reply:
Depends on the use cases, as I asked previously : 
Are only values symbolic ?
Or do we want environment variable names to be symbolic as well ?